docs/ntppool/en/use.html

```diff
@@ -1,77 +1,102 @@
-	</p>
-
-	[% INCLUDE "ntppool/use/sample-ntpq.html" %]
-
-	<p>
-	The IP addresses will be different, because you've been assigned random
-	timeservers. The essential thing is that one of the lines starts with an
-	asterisk (<code>*</code>), this means your computer gets the time from the internet
-	- you'll never have to worry about it again!
-	</p>
-	<p>
-	Looking up <code>pool.ntp.org</code> (or <code>0.pool.ntp.org</code>,
-	<code>1.pool.ntp.org</code>, etc) will usually return IP addresses for servers
-	in or close to your country. For most users this will give the best results.
-	</p>
-
-	<p>You can also use the <a href="/zone/@">continental zones</a> (For example
-	<a href="/zone/europe">europe</a>,
-	<a href="/zone/north-america">north-america</a>,
-	<a href="/zone/oceania">oceania</a>
-	or <a href="/zone/asia">asia</a>.pool.ntp.org),
-	and a country zone (like
-	ch.pool.ntp.org in Switzerland) - for all these zones, you can again use the 0,
-	1 or 2 prefixes, like 0.ch.pool.ntp.org.  Note, however, that the country zone
-	might not exist for your country, or might contain only one or two timeservers.
-	</p>
-	<p>
-	If you're using <b>a recent Windows version</b>, you can use the ntp
-	client that is built into the system. As administrator enter</p>
+    <h3 id="use">How do I use pool.ntp.org?</h3>
+
+    <p>
+    If you just want to synchronise your computers clock to the network, the configuration file (for the ntpd program from the <a href="http://www.ntp.org">ntp.org distribution</a>, on any supported operating system - <b>Linux, *BSD, Windows and even some more exotic systems</b>) is really simple:
+    </p>
+
+    [% INCLUDE "ntppool/use/sample-config.html" %]
+
+    <p>
+    The 0, 1, 2 and 3.pool.ntp.org names point to a random set of servers that will
+    change every hour.  Make sure your computer's clock is set to something
+    sensible (within a few minutes of the 'true' time) - you could use <code>ntpdate
+    2.pool.ntp.org</code>, or you could just use the <code>date</code> command and set it
+    to your wristwatch. Start ntpd, and after some time (this could take as long as
+    half an hour!), <code>ntpq -pn</code> should output something like:
+    </p>
+
+    [% INCLUDE "ntppool/use/sample-pool-ntpq.html" %]
+
+    <p>
+    The IP addresses will be different, because you've been assigned random
+    timeservers. The essential thing is that one of the lines starts with an
+    asterisk (<code>*</code>), this means your computer gets the time from the internet
+    - you'll never have to worry about it again!
+    </p>
+    <p>On more recent Linux operating systems, time setting has been delegated to
+    <code>systemd</code>. You can use <code>timedatectl</code> to set the time:
+    </p>
+
+    [% INCLUDE "ntppool/use/sample-timedatectl.html" %]
+
+    <p>
+    On RedHat et al. (Fedora, CentOS, etc.) <a href="https://chrony.tuxfamily.org/"><code>chronyd</code></a>
```

Comment:
It's also the default on Ubuntu & similar, and I suspect SUSE probably has made a similar change.

```suggestion
    On some Linux distributions <a href="https://chrony.tuxfamily.org/"><code>chronyd</code></a>
```

Reply:
I was thinking about reverting the second commit and then applying your suggestion. But that would result in a conflict, right? It's easy enough to resolve or apply manually after the revert. But what's the GitHub way of going about it?